plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMergeSink.java

```diff
@@ -324,7 +324,7 @@ private ReaderPageSource createParquetPageSource(Path path)
 
         private FileDeletion(List<String> partitionValues)
         {
-            this.partitionValues = ImmutableList.copyOf(requireNonNull(partitionValues, "partitionValues is null"));
+            this.partitionValues = new ArrayList<>(requireNonNull(partitionValues, "partitionValues is null"));
```

Comment:
Is this meant to be mutable? If not, wrap it in plain `Collections#unmodifiableList` with a comment why `ImmutableList` cannot be used.

Reply:
Good idea; added.